pkg/cmd/root.go

```diff
@@ -57,17 +56,32 @@ func Execute() (*types.CmdValues, error) {
 
 func addProviderFlags(cmd *cobra.Command, vals *types.CmdValues) *cobra.Command {
 
-	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", []string{"docker-compose.yml"}, "Provide docker-compose files, comma separated.")
+	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", nil, "Provide docker-compose files, comma separated. Takes docker-compose.yml or docker-compose.yaml if no file is specified.")
```

Comment:
The usage message better can be made 

`Provide comma separated docker-compose files. Default: 'docker-compose.yml' or 'docker-compose.yaml'`